### DISCUSSION TITLE
Dropbox links aren't working on Hubs Cloud

### DISCUSSION THREAD

**Discussion by VirtualConferenceHubs** [mozilla/hubs#2864] [Hubs Cloud Questions] [answered]
Hi, is there a known/ongoing issue related to using dropbox links in hubs cloud ? None of the links from dropbox are working for me.
![dropbox](https://user-images.githubusercontent.com/66517679/90308376-04e9fd80-ded7-11ea-863a-925079f3dcf8.PNG)

**Comment by robinkwilson** [comment, marked as answer]
We have an open issue for this **Dropbox files are not supported in Hubs Cloud, but work on HMC**: https://github.com/mozilla/hubs-cloud/issues/73
